src/components/FormInput/style.js

```diff
@@ -1,8 +1,9 @@
 import styled from 'styled-components'
 
-export const FormInputWrapper = styled.label`
-    color: ${props => (props.isFunny ? 'red' : '')};
-    @media only screen and (max-width: 850px) {
-        width: 100%;
-    }
+const FormInputWrapper = styled.label`
+  color: ${props => (props.isFunny ? 'red' : '')};
```

Comment:
for empty values it's better if you used it like this
`color: ${props => (props.isFunny ? 'red' : undefined)};`
undefined is preferred here because the value undefined on a property is like eliminating the property. Contrary empty string will produce a property with value ''.